app/api/schema/speakers.py

```diff
@@ -12,6 +13,13 @@ class SpeakerSchema(SoftDeletionSchema):
     Speaker Schema based on Speaker Model
     """
 
+    @validates_schema(pass_original=True)
+    def validate_json(self, data, original_data):
+        if data.get('complex_field_values'):
+            if any(isinstance(i, dict) for i in data['complex_field_values'].values()):
```

Comment:
```json
{
  "a": [{... big nested data ...}, {... big nested data ...}, {... big nested data ...}] 
}
```

This will be supported

Reply:
Also, let's add a limit to the fields in settings. By default, allow only 30 items. I think it is fair. If more are required, then it can be changed by the admin

Reply:
> ```
> {
>   "a": [{... big nested data ...}, {... big nested data ...}, {... big nested data ...}] 
> }
> ```
> 
> This will be supported

@iamareebjamal for this can we impose that the value **has** to be a string or number or null, that will eliminate the possibility of a list or other unsupported types.

Reply:
Agree